client/blocks/inline-help/popover.jsx

```diff
@@ -167,19 +199,57 @@ class InlineHelpPopover extends Component {
 	}
 }
 
+const optOut = siteId => {
+	return withAnalytics(
+		composeAnalytics(
+			recordGoogleEvent(
+				'Gutenberg Opt-Out',
+				'Clicked "Switch to the classic editor" in the help popover.',
+				'Opt-In',
+				false
+			),
+			recordTracksEvent( 'calypso_gutenberg_opt_in', {
+				opt_in: false,
+			} ),
+			bumpStat( 'gutenberg-opt-in', 'Calypso Help Opt Out' )
+		),
+		setSelectedEditor( siteId, 'classic' )
+	);
+};
+
+const redirect = classicUrl => {
+	return navigate( classicUrl );
+};
+
+const swapHistory = classicUrl => {
+	return replaceHistory( classicUrl );
+};
```

Comment:
are these two functions adding any benefit to us over passing in `navigate` and `replaceHistory` directly? seems like we could get away with using them as-is for now

Reply:
Thanks for this! I refactored and removed these unneeded wrapper functions.

Tested and still works as expected.